--- a/serve/settings.py	2015-03-19 00:40:02.253528784 +0000
+++ b/serve/settings.py	2015-03-19 00:42:52.749530743 +0000
@@ -36,6 +36,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'timelog',
 )
 
 MIDDLEWARE_CLASSES = (
@@ -58,8 +59,11 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'timelog',
+        'USER': 'timelog',
+        'PASSWORD': '',
+        'HOST': '',
     }
 }
 
