--- a/serve/settings.py	2015-04-02 09:06:14.253530738 +0000
+++ b/serve/settings.py	2015-04-02 09:06:41.093531581 +0000
@@ -37,6 +37,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'timelog',
 )
 
 MIDDLEWARE_CLASSES = (
@@ -76,8 +77,11 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'timelog',
+        'USER': 'timelog',
+        'PASSWORD': '',
+        'HOST': '',
     }
 }
 
